### PR TITLE
Add Red Hat to list of production users [skip ci]

### PR DESCRIPTION
**Pull request by mbrukman**
RedHat built their application migration and assessment tool with an
open-source project they released (Windup), which directly depends on
JanusGraph.

Partially addresses issue #1153; will be fixed by the corresponding
website update to add the logo.

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [/] Have you written and/or updated unit tests to verify your changes?
- [/] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [/] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [/] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered?
- [x] If this PR is a documentation-only change, have you added a `[skip ci]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ The following users have deployed JanusGraph in production.
 * [Netflix](https://www.netflix.com) -
   [video](https://youtu.be/KSmAdtMJYEo?t=1h2m17s) and
   [slides](https://www.slideshare.net/RoopaTangirala/polyglot-persistence-netflix-cde-meetup-90955706) (graph discussion starts at #86)
+* [RedHat](https://www.redhat.com/) - [application migration and assessment tool](https://developers.redhat.com/products/rhamt/overview/) built on [Windup](https://github.com/windup/windup)
```
Comment (jsight) [inline README.md]: This looks good, but could you please change it to "Red Hat" (with a space between the two words)? Thanks. :)

**Comment by jsight** [comment]
Just one minor comment, otherwise it looks great to me!

File: README.md (hunk 1 above)
Comment (mbrukman) [reply README.md]: Fixed! Sorry about that, didn't realize it's actually two words. Will know for the future. :-)

Comment (jsight) [reply README.md]: Almost everyone makes that mistake, due to the font. Thanks for fixing it!

**Comment by mbrukman** [comment]
@amcp, @jerryjch — please also look at this change, as a companion to https://github.com/JanusGraph/janusgraph.org/pull/47

**Pull request merged**